enterprise/api_client/lms.py

```diff
@@ -390,8 +386,9 @@ def enroll_user_in_course_locally(user, course_id, mode):
     use to create a callback. Once we're able to depend on having Django 1.9, we
     can shift over to that, but for right now, we have to do it this way.
     """
-    if CourseKey is None and CourseEnrollment is None:
+    if CourseEnrollment is None:
```

Comment:
Has something to do with the `from opaque_keys.edx.keys import CourseKey` above -- just ran into this and this change fixes test failures.